webui/react/src/stores/cluster.tsx

```diff
@@ -183,6 +186,9 @@ class ClusterStore extends PollingStore {
     getResourcePools({}, { signal: signal ?? canceler.signal })
       .then((response) => {
         this.#resourcePools.set(Loaded(response));
+        this.#multiResourceManagers.set(
+          Loaded(uniq(response.map((p) => p.resourceManagerName)).length > 1),
```

Comment:
this is derived from the `resourcePools` part of the response -- expose this info as a selection instead. Furthermore, `uniq` is unnecessary -- just put it in a set and get the resulting size:

```tsxpublic readonly multiResourceManagers = this.#resourcePools.select((resourcePools) =>
  Loadable.map(resourcePools, (r) => new Set(r.map((p) => p.resourceManagerName)).length > 1))
```